test/client-src/default/utils/getCurrentScriptSource.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
Let's rename test directory to `client` not `client-src`. `src` postfix used because we build bundle